api/types.go

```diff
@@ -231,17 +231,23 @@ type Options struct {
 
 // Runner options which must be set when the model is loaded into memory
 type Runner struct {
-	NumCtx    int   `json:"num_ctx,omitempty"`
```

Comment:
Did you mean to make all the whitespace changes in this file?
The 7 additions in this file are obfuscated by whitespace changes.
It is possible to configure GitHub or git tools to hide whitespace changes but wouldn't a clean diff be better?
Very excited by the feature BTW!
@bfroemel